packages/magento-recently-viewed-products/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@graphcommerce/magento-recently-viewed-products",
+  "homepage": "https://www.graphcommerce.org/",
+  "repository": "github:graphcommerce-org/graphcommerce",
+  "version": "7.1.0-canary.8",
+  "sideEffects": false,
+  "prettier": "@graphcommerce/prettier-config-pwa",
+  "eslintConfig": {
+    "extends": "@graphcommerce/eslint-config-pwa",
+    "parserOptions": {
+      "project": "./tsconfig.json"
+    }
+  },
+  "dependencies": {
+    "@graphcommerce/graphql": "7.1.0-canary.8",
+    "@graphcommerce/graphql-mesh": "7.1.0-canary.8",
```

Comment:
I don't think that graphql-mesh is required?

Reply:
graphql-mesh is used in the generated `RecentlyViewedProducts.gql.ts` file. Does that not make it a dependency?